### PR TITLE
DAMD-96: Index fibers from multiple spectrographs in PfsDesign

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: python/pfs/instmodel/splinedPsf.py [hunk 1]
```diff
@@ -18,11 +18,13 @@
 import scipy.ndimage.interpolation
 
 from . import spotgames
-from .spectrum import LineSpectrum, ArcSpectrum
+from .slit import Slit
 
 from . import psf
 reload(psf)
 
+NUM_FIBERS = 651  # Number of fibers per spectrograph
```
Comment (hassanxp) [inline python/pfs/instmodel/splinedPsf.py:26]: Only used once in this file. Shouldn't this constant be used also for the fiber numbering in ``slit.py``?

**Review by hassanxp — APPROVED** [review]
Minor comments, but changes are fine by me.

File: python/pfs/instmodel/splinedPsf.py (hunk 1 above)
Comment (PaulPrice) [reply python/pfs/instmodel/splinedPsf.py:26]: Sorry, neglected to address this before merging. I'll fix it up.

File: python/pfs/instmodel/slit.py [hunk 2]
```diff
@@ -58,8 +58,9 @@ class Slit(object):
 
     slits = slit1, slit1, slit2, slit2
 
-    def __init__(self, slitId):
-        self.fiber0 = 1
+    def __init__(self, spectrograph):
```
[the remark this answers is not in the document]
Comment (PaulPrice) [reply python/pfs/instmodel/slit.py:61]: There never has been. This should be fixed as part of the general simulator cleanup ticket.

**Pull request merged**